tasks/test_module/my_task.py

```diff
@@ -0,0 +1,417 @@
+            ("the_geom", cusec_geom),
+        ])
+
+class SourceTags(TagsTask):
+    def version(self):
+        return 1
+
+    def tags(self):
+        return [
+            OBSTag(
+                id='ine-source',
+                name='National Statistics Institute (INE) Legal Notice',
+                type='source',
+                description='More information `here <http://ine.es/ss/Satellite?L=1&c=Page&cid=1254735849170&p=1254735849170&pagename=Ayuda%2FINELayout#>`_')
+        ]
+
+class LicenseTags(TagsTask):
+    def version(self):
+        return 1
+
+    def tags(self):
+        return [OBSTag(id='ine-license',
+                name='National Statistics Institute (INE)',
+                type='license',
+                description='`INE website <http://www.ine.es>`_')
+            ]
+
+class Geometry(TableTask):
+
+    def version(self):
+        return 1
+
+    def requires(self):
+        return {
+            'meta': GeometryColumns(),
+            'data': RawGeometry()
+        }
+
+    def columns(self):
+        return self.input()['meta']
+
+    def timespan(self):
+        # TODO Add a param year
+        return '2011'
+
+    def populate(self):
+        session = current_session()
+        session.execute('INSERT INTO {output} '
+                        'SELECT cusec as cusec_id, '
+                        '       wkb_geometry as cusec_geom '
+                        'FROM {input} '.format(
+                            output=self.output().table,
+                            input=self.input()['data'].table))
+
+class SeccionColumns(ColumnsTask):
+
+    # metadata source: http://www.ine.es/en/censos2011_datos/indicadores_seccen_rejilla_en.xls
+
+    def requires(self):
+        return {
+            'tags': SubsectionTags(),
+            'sections': SectionTags(),
+            'units': UnitTags(),
+            'license': LicenseTags(),
+            'source': SourceTags(),
+        }
+
+    def version(self):
+        return 1
+
+    def columns(self):
+        input_ = self.input()
+        spain = input_['sections']['spain']
+        tags = input_['tags']
+        units = input_['units']
+        source = input_['source']['ine-source']
+        license = input_['license']['ine-license']
+
+        total_pop = OBSColumn(
+            id='t1_1',
+            name='Total population',
+            description='',
+            aggregate='sum',
+            type='Numeric',
+            weight=5,
+            tags=[tags['age_gender'], spain, units['people']])
+        male_pop = OBSColumn(
+            id='t2_1',
+            name='Males',
+            description='',
+            aggregate='sum',
+            type='Numeric',
+            weight=5,
+            tags=[tags['age_gender'], spain, units['people']],
+            targets={total_pop: DENOMINATOR})
+
+        columns = OrderedDict([
+            ('total_pop', total_pop),
+            ('male_pop', male_pop),
+            ])
+
+        for _, col in columns.iteritems():
```

Comment:
Instead of adding this iteratively, can contain them in one fell swoop by overriding the [.tags](https://github.com/CartoDB/bigmetadata/blob/master/tasks/util.py#L976) method.